README.md

```diff
@@ -5,6 +5,8 @@
 # gonomics
 A collection of genomics software tools written in Go (golang).
 
+![GonomicsReleaseFigure](https://github.com/vertgenlab/gonomics/assets/50707122/cf62dc98-1975-4d03-aabf-dcf4096def07)
```

Comment:
I am having trouble telling where this file is stored.

Reply:
You can preview it how the README looks here: https://github.com/vertgenlab/gonomics/tree/rimangan-patch-1

Reply:
That preview looks good, but I am wondering where the file is stored.  I don't see it being added as part of this PR.  Was it already added as part of a previous PR?

Reply:
Looks like the image is uploaded to
- https://user-images.githubusercontent.com/50707122/253075286-cf62dc98-1975-4d03-aabf-dcf4096def07.png

I've only done this in PR comments and never added an actual image to a readme file, but it looks like if you edit the  readme file in the internet browser by clicking on the pencil, you can drag an image and it'll automatically upload to github's aws bucket. Here's a screen recording of my performing the action:

https://github.com/vertgenlab/gonomics/assets/11465271/5ade149a-db13-49bc-99a6-6a4748850b7a